lib/async_hooks.js

```diff
@@ -60,19 +60,11 @@ async_wrap.setupHooks({ init,
                         destroy: emitDestroyN });
 
 // Used to fatally abort the process if a callback throws.
-function fatalError(e) {
```

Comment:
Letting an error bubble with `try {} finally {}` have always been the standard way of handling throws.